app.js

```diff
@@ -54,6 +59,38 @@ app.use(function(req, res, next) {
    }
 });
 
+app.use(cookieParser());
+app.use(session({
+        secret:'secret_token',
+        cookie: {maxAge: 24*3600*1000},//cookie max age set to one day
+        resave: true,
+        saveUninitialized: true
+        }));
+
+//if username and password supplied enabled auth
+var seedUsername = nconf.get('USERNAME')
+var seedPassword = nconf.get('PASSWORD')
+if(seedUsername && seedUsername !== "" && seedPassword && seedPassword !== "") {
+    app.use('/login', loginRoutes);
+    app.use('/logout', logoutRoutes);
+
+    //routes registered below this filter will require a valid session value/user
+    app.all('*',function(req,res,next){
+        res.header("Access-Control-Allow-Origin", req.headers['origin']);
```

Comment:
Why the CORS headers? I don't expect to be showing the login form from another domain.